descartes_rpa/fetch/descartes.py

```diff
@@ -0,0 +1,23 @@
+import requests
+
+
+def fetch_descartes_tissue(tissue: str, out_file: str) -> None:
+    """Function to fetch Loom Single-Cell tissue data from
+    Descartes human database.
+
+    Args:
+        tissue: Name of the tissue that is going to be downloaded
+        out_file: Output file that is going to store .loom data
+
+    Examples:
```

Comment:
lets only pass in the output file path here. since there is only one loom file and maybe change function name to fetch_descartes_human_tissue. 

Don't think we could use the same function on mouse data (if time allows)

thanks :)

Reply:
Done!